stream-chat-android/src/main/java/com/getstream/sdk/chat/adapter/ChannelItemPayloadDiff.kt

```diff
@@ -6,6 +6,7 @@ public data class ChannelItemPayloadDiff(
     val lastMessage: Boolean,
     val lastMessageDate: Boolean,
     val readState: Boolean,
+    val unreadCount: Boolean,
```

Comment:
Should we also update [this code](https://github.com/GetStream/stream-chat-android/blob/2006964e2cd5149f2be92ab28d5de358d0c034df/stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/Channel.kt#L51-L57)?

Reply:
I think the difference is that in the old diff we only check if current user read changed while in the new diff this case is covered by the fact we calculate the diff for all `ChannelUserRead` objects.

Reply:
However, will recheck it